src/vm/amd64/JitHelpers_InlineGetThread.asm

```diff
@@ -147,6 +147,11 @@ align 16
         jmp     JIT_Box
 NESTED_END JIT_BoxFastMP_InlineGetThread, _TEXT
 
+RELATIVE_PTR_LOAD macro Reg, BaseReg, Offset
```

Comment:
There should not be a new argument to the allocation helpers. The templateMT argument should be passed to the allocation helpers instead of the current ArrayTypeDesc argument.